client.go

```diff
@@ -41,48 +41,56 @@ type SingleServerClient struct {
 	isLocalTest bool
 }
 
-func NewClientImpl(deployFile string, targetName string, isLocalTest bool) (*SingleServerClient, error) {
+// Client which communicates with multiple underlying servers at once. Used if
+// the target is actually a group of servers
+type MultiServerClient struct {
```

Comment:
Sorry, I forgot "Client" is the name of the interface.  Perhaps still use the term "Target" instead of "Server" everywhere.